timeless/employees/models.py

```diff
@@ -37,5 +32,20 @@ class Employee(TimestampsMixin, DB.Model):
 
     company = DB.relationship("Company", back_populates="employees")
 
+    def __init__(self, **kwargs):
+        super(Employee, self).__init__(**kwargs)
+        if "password" in kwargs:
+            self.password = bcrypt_sha256.hash(kwargs.get("password"))
+        if "pin_code" in kwargs:
```

Comment:
@bkuzmic What happens if pin code is not provided?

Reply:
@emilianodellacasa Should I throw KeyError? Also, should I do the same for other non optional parameters, like username? The question is more: do we want to have this kind of validation on the model side or in the controller?

Reply:
@bkuzmic Supposedly, the pin code should be assigned randomly. I am ok with having an optional parameter, but the main request was to assign a random value to it;

Reply:
@emilianodellacasa Ok, I'll generate a random pin number in the constructor.